src/javascript/app/pages/user/reality_check/reality_check.ui.js

```diff
@@ -122,6 +122,7 @@ const RealityCheckUI = (() => {
             BinarySocket.send({ reality_check: 1 }).then((response) => {
                 if (response.error && !/user\/statementws\.html/.test(window.location.pathname)) { // don't block statement page for reality check error, but block all other pages
                     $('#content').empty().html($('<div/>', { class: 'container' }).append($('<p/>', { class: 'notice-msg center-text', text: response.error.message })));
+                    window.location.reload();
```

Comment:
won't we get stuck in an endless loop here? e.g. if server is down etc it will keep reloading the page on user's behalf

Reply:
Noted, I'll add a condition to check if token is still valid / exists before triggering the reload.

Reply:
I'll revert it for now since we want to do a release today